### ISSUE TITLE
Chapter 8 p.425 Q4 Typo

### ISSUE THREAD

**Issue by jwhitt3r** [head-first-csharp/fourth-edition#33]
I believe there is a typo in the answer for Question: Ok, that explains the "collection" part, But what makes it "generic"?

"... items that were packaged in white wrapper with black **type**..."

Should this be black tape?

**Comment by andrewstellman** [comment]
Yep, it should! Thanks!

**Comment by andrewstellman** [comment]
Nope, black type! When I was a kid back in the 80s you could walk down a supermarket aisle and see a bunch of products that looked like this. 

![Generic products in a supermarket](https://raw.githubusercontent.com/andrewstellman/random-assets/main/indiciadesign.com_.jpg)
